assembly/src/ast/nodes/advice.rs

```diff
@@ -26,6 +26,7 @@ pub enum AdviceInjectorNode {
     InsertMem,
     InsertHdword,
     InsertHdwordImm { domain: u8 },
+    FalconSign,
```

Comment:
I would probably make the name much more "informative". For example, something like `SignRpoFalcon512`. Another option (if we introduce `SignatureKind` as mentioned above) is to do it like `PushSignature { kind: SignatureKind }`.